langserver/definition.go

```diff
@@ -39,6 +41,24 @@ func (h *LangHandler) handleDefinition(ctx context.Context, conn jsonrpc2.JSONRP
 	return locs, nil
 }
 
+func (h *LangHandler) handleTypeDefinition(ctx context.Context, conn jsonrpc2.JSONRPC2, req *jsonrpc2.Request, params lsp.TextDocumentPositionParams) ([]lsp.Location, error) {
+	// note the omission of Godef case; don't want to try to
+	// handle two different ways of doing this just yet.
```

Comment:
yeah leave out godef support. We haven't investigated yet, but go's cache dir may enable us to remove godef based lookup.